permissions/DataStorages.permission

```diff
@@ -5,3 +5,9 @@
 # x-sailjail-description = Removable media
 # x-sailjail-translation-key-long-description =
 # x-sailjail-long-description = Use memory cards and USB sticks
+
+### Allow to listen/talk to udisks2
+# Used for mounting, formatting
+include /etc/sailjail/permissions/UDisks.permission
+
+ignore disable-mnt
```

Comment:
Wondering what else gets exposed with this other than external storages?

Reply:
No need to wonder, [the manual pages ](https://firejail.wordpress.com/features-3/man-firejail-profile/) just state this clearly:
`
       disable-mnt
              Disable /mnt, /media, /run/mount and /run/media access.
`

Unless the curtains do not match the drapes?